src/main/java/ch/njol/skript/effects/EffForceAttack.java

```diff
@@ -1,67 +1,117 @@
 package ch.njol.skript.effects;
 
-import org.bukkit.entity.Entity;
-import org.bukkit.entity.LivingEntity;
-import org.bukkit.event.Event;
-import org.jetbrains.annotations.Nullable;
-
 import ch.njol.skript.Skript;
-import ch.njol.skript.doc.Description;
-import ch.njol.skript.doc.Examples;
-import ch.njol.skript.doc.Name;
-import ch.njol.skript.doc.RequiredPlugins;
-import ch.njol.skript.doc.Since;
+import ch.njol.skript.config.Node;
+import ch.njol.skript.doc.*;
 import ch.njol.skript.lang.Effect;
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
 import ch.njol.util.Kleenean;
+import org.bukkit.entity.Damageable;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.log.runtime.SyntaxRuntimeErrorProducer;
 
 @Name("Force Attack")
-@Description("Makes a living entity attack an entity with a melee attack.")
+@Description({
+	"Makes a living entity attack an entity with a melee attack.",
+	"Using 'attack' will make the attacker use the item in their main hand "
+		+ "and will apply extra data from the item, including enchantments and attributes.",
+	"Using 'damage' with a number of hearts will not account for the item in the main hand "
+		+ "and will always be the number provided."
+})
+@Example("""
+	spawn a wolf at location(0, 0, 0)
+	make last spawned wolf attack all players
+	""")
+@Example("""
+	spawn a zombie at location(0, 0, 0)
+	make player damage last spawned zombie by 2
+	""")
 @Examples({"spawn a wolf at player's location",
 	"make last spawned wolf attack player"})
-@Since("2.5.1")
+@Since("2.5.1, INSERT VERSION (multiple, amount)")
 @RequiredPlugins("Minecraft 1.15.2+")
-public class EffForceAttack extends Effect {
+public class EffForceAttack extends Effect implements SyntaxRuntimeErrorProducer {
 	
 	static {
 		Skript.registerEffect(EffForceAttack.class,
-			"make %livingentities% attack %entity%",
-			"force %livingentities% to attack %entity%");
+			"make %livingentities% attack %entities%",
+			"force %livingentities% to attack %entities%",
+			"make %livingentities% damage %entities% by %number% [heart[s]]",
+			"force %livingentities% to damage %entities% by %number% [heart[s]]");
```

Comment:
Do you think omitting `hearts` should have a different behavior (not multiplying by 2)?